pkg/operator/ceph/cluster/mon/health_test.go

```diff
@@ -102,39 +102,34 @@ func TestCheckHealthNotFound(t *testing.T) {
 	c.waitForStart = false
 	defer os.RemoveAll(c.context.ConfigDir)
 
-	c.mapping.Node["rook-ceph-mon1"] = &NodeInfo{
-		Name:    "node0",
-		Address: "",
+	c.mapping.Node["a"] = &NodeInfo{
+		Name: "node0",
 	}
-	c.mapping.Node["rook-ceph-mon2"] = &NodeInfo{
-		Name:    "node0",
-		Address: "",
+	c.mapping.Node["b"] = &NodeInfo{
+		Name: "node0",
 	}
 	c.mapping.Port["node0"] = cephmon.DefaultPort
-	c.maxMonID = 10
+	c.maxMonID = 4
 
 	c.saveMonConfig()
 
 	// Check if the two mons are found in the configmap
 	cm, err := c.context.Clientset.CoreV1().ConfigMaps(c.Namespace).Get(EndpointConfigMapName, metav1.GetOptions{})
 	assert.Nil(t, err)
-	if cm.Data[EndpointDataKey] == "rook-ceph-mon1=1.2.3.1:6790,rook-ceph-mon2=1.2.3.2:6790" {
-		assert.Equal(t, "rook-ceph-mon1=1.2.3.1:6790,rook-ceph-mon2=1.2.3.2:6790", cm.Data[EndpointDataKey])
-	} else {
-		assert.Equal(t, "rook-ceph-mon2=1.2.3.2:6790,rook-ceph-mon1=1.2.3.1:6790", cm.Data[EndpointDataKey])
+	if cm.Data[EndpointDataKey] != "a=1.2.3.1:6790,b=1.2.3.2:6790" {
```

Comment:
You need to add a `else` as previously as the test won't "fail" nor report an error.

Reply:
There are two cases here. Either it equals the statement in the conditional or it equals the the conditional in the assert. If it's neither of those, the assert will fail. So I don't think we actually need an `else` here.